certs/certs.c

```diff
@@ -28,7 +28,10 @@ const char certificates[] =
 "ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\r\n"
 "R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*DigiCert Global Root CA*/
+#ifndef LEAN_CERTS
```

Comment:
This should likely be documented somewhere if this gets pushed. Otherwise people will have to stumble upon this to use it.

Reply:
This should also be tested at the gate

---
In reply to: [337152560](https://github.com/Azure/azure-iot-sdk-c/pull/1274#discussion_r337152560) [](ancestors = 337152560)